server/graphql/v2/input/PaymentMethodInput.ts

```diff
@@ -75,7 +75,7 @@ export const getLegacyPaymentMethodFromPaymentMethodInput = async (
   if (!pm) {
     return null;
   } else if (pm.id) {
-    return fetchPaymentMethodWithReference(pm);
+    return fetchPaymentMethodWithReference(pm) as any;
   }
```

Comment:
Maybe we should rather change the return type for this one to something like `Promise<Record<string, unknown> & {service: string; type: string;}>`. Not a fan of returning `any`, but not a blocker if it's too much complexity for now.

Reply:
Agreed. I think we should eventually go over the linter warnings for any and remove as much as we can.